SpatialGDK/Extras/core-sdk.version

```diff
@@ -1,2 +1,2 @@
-15.0.1
+15.1.0-preview-1

```

Comment:
perhaps we revert this?

Reply:
Will update with 15.1.0 when the full release is out